cilantro_audit/widgets/view_audit_templates.kv

```diff
@@ -0,0 +1,110 @@
+#:kivy 1.11.1
+
+
+Manager:
+
+
+<ThisPage>:
+    name: 'this_page'
+
+    BoxLayout:
+        orientation: 'vertical'
+
+        BoxLayout:
+            name: 'header'
+            size_hint: 1, 0.1
+
+            orientation: 'horizontal'
+
+            canvas.before:
+                Color:
+                    rgb: 0.275, 0.314, 0.275, 1
+                Rectangle:
+                    size: self.size
+                    pos: self.pos
+
+            Button:
+                text: 'Back'
+                size_hint: 0.3, 1.0
+                on_release:
+                    app.root.current = 'AuditorPage'
```

Comment:
If we use the constants, this becomes a type and results in a `ScreenManagerException` where its looking for `AuditorPage` but we have defined `AUDITOR_SCREEN` as `AuditorScreen`. 
Changing to `app.root.current = AuditorScreen` fixes.

Reply:
Done.